src/main/java/org/osiam/resources/helper/AttributesRemovalHelper.java

```diff
@@ -135,22 +152,21 @@
     }
 
     private ObjectWriter getObjectWriter(List<String> fieldsToReturn) {
+        if (fieldsToReturn.isEmpty()) {
+            FilterProvider filter = new SimpleFilterProvider().addFilter("filter properties by name", SimpleBeanPropertyFilter.serializeAll());
```

Comment:
Do we really need another no-op filter? Shouldn't the one defined in the global `ObjectMapper` be enough?